Sort by primary keys by default

sagargg opened the pull request.

sagargg committed:
[order][xs] sort by primary keys by default

anuveyatsu committed:
[order rows][xs]: improved the logic so that it still works if no primary keys are given.

anuveyatsu commented:
@sagargg tests were failing so I can't approve this PR.

I have also added a commit, please check.

anuveyatsu committed:
[tests][s]: fixed issue with canvas in jsdom by installing 'jest-canvas-mock' lib.

Also, now we use 'replace' with regex instead of 'replaceAll' as it's not yet implemented in node (v14) and on some browsers.
